autoload/lsp/lspserver.vim

```diff
@@ -261,6 +261,7 @@ def SendMessage(lspserver: dict<any>, content: dict<any>): void
     return
   endif
   ch->ch_sendexpr(content)
+  util.TraceLog(false, $'Sent: {content->string()}')
```

Comment:
This will result in too many messages in the trace file (as the plugin constantly sends buffer updates to the server).  We need to exclude the "textDocument/didChange" message from this trace.

Reply:
What about to check for req.id? If I'm not wrong, it is added only to messages for which a response is expected.